src/proto/grpc/status/README

```diff
@@ -0,0 +1,2 @@
+The status.proto file is copied from
+https://github.com/googleapis/googleapis/blob/master/google/rpc/status.proto.
```

Comment:
Do we want to add some sort of build-time check that we have the latest version of the file?